.changelog/12444.txt

```diff
@@ -0,0 +1,7 @@
+```release-note:bug
+compare node names case insensitively in more places
```

Comment:
```suggestion
catalog: compare node names case insensitively in more places
```

Maybe this is an appropriate component. It does apply more generally but I think this could be close enough.